src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
+const seven = 7;
+const three = 3;
+const saleForThreeDays = 20;
+const saleForSevenDays = 50;
```

Comment:
These variable names (`seven`, `three`, `saleForThreeDays`, `saleForSevenDays`) are not descriptive enough. It's better to use names that reflect the purpose of the variable. For example, `LONG_TERM_RENTAL`, `SHORT_TERM_RENTAL`, `LONG_TERM_DISCOUNT`, `SHORT_TERM_DISCOUNT`.